### ISSUE TITLE
Run Experiemnts to promote "screenshot" feature 

### ISSUE THREAD

**Issue by joechengla** [mozilla-tw/Rocket#1422]
and understand its impact on retention

**Comment by chianger** [comment]
I recalled that from a offline discussion that we're going to drop this out of 1.0.5

**Comment by joechengla** [comment]
yes let's move this out as the screenshot feature will be very obvious in 2.0

**Comment by joechengla** [comment]
now with Firebase, we can either use FCM or other methods through remote config
